web/urls.py

```
@@ -3,13 +3,16 @@
 from django.contrib import admin
 from django.contrib.auth.decorators import login_required
 from django.urls import include, path
+from django.urls import path, include
```

Ruff (0.8.2)

Redefinition of unused `path` from line 5

Remove definition: `path`

(F811)